changes.d/fix.6170.md

```diff
@@ -0,0 +1 @@
+Fix an issue where the Cylc logo could appear in the workflow log.
```

Comment:
Appear twice?

Reply:
It could appear twice in the user's terminal, but only once in the workflow log.

The logo shouldn't have appeared in the workflow log in the first place.